src/main/java/hu/dpc/rt/kafkastreamer/importer/ElasticsearchClient.java

```diff
@@ -85,7 +85,12 @@ public void index(JSONObject record) {
         if (metrics == null) {
             metrics = new ElasticsearchMetrics(record.getInt("partitionId"));
         }
+        if(record.has("value.processInstanceKey")){
```

Comment:
@ankita10r Is this tested? Are you sure this is a nullsafe way of soing it?

Reply:
Requesting for a revert on commit, sending updated PR